cmd/build.go

```diff
@@ -38,15 +39,48 @@ func Build(ctx context.Context) *cobra.Command {
 	options := build.BuildOptions{}
 	cmd := &cobra.Command{
 		Use:   "build [PATH]",
-		Args:  utils.BuildMaximumNArgsAccepted(1, "https://okteto.com/docs/reference/cli/#build"),
 		Short: "Build (and optionally push) a Docker image",
 		RunE: func(cmd *cobra.Command, args []string) error {
 
-			if err := contextCMD.NewContextCommand().Run(ctx, &contextCMD.ContextOptions{}); err != nil {
```

Comment:
I think we need to run this to setup secrets and other information

Reply:
Nevermid I just saw its removing duplicated code

Reply:
secrets is not supported as build param at the manifest, the secrets are allowed as flag if single build is done.